shared/js/ui/templates/shared/tracker-network-icon.es6.js

```diff
@@ -0,0 +1,13 @@
+const bel = require('bel')
+
+module.exports = function (siteRating, isWhitelisted) {
+    let iconNameModifier = 'blocked'
+
+    if (isWhitelisted && (siteRating.before === "D")) {
+            iconNameModifier = 'warning'
+    }
+
+    let iconName = 'major-networks-' + iconNameModifier
```

Comment:
Technically this could be a `const`, which we've been using for variables that don't change (let is for variables that do get modified).